packages/apps/docs/src/components/Layout/Full/styles.css.ts

```diff
@@ -0,0 +1,61 @@
+import { breakpoints, sprinkles, vars } from '@kadena/react-ui/theme';
+
+import { $$pageWidth } from '../global.css';
+
+import { createVar, style } from '@vanilla-extract/css';
+
+export const $$shadowWidth = createVar();
+export const $$asideMenuWidthCode = createVar();
+export const $$asideMenuWidthMDDefault = createVar();
+export const $$asideMenuWidthLGDefault = createVar();
+
+export const asidebackgroundClass = style([
+  sprinkles({
+    display: 'none',
+  }),
+
+  {
+    vars: {
+      [$$shadowWidth]: vars.sizes.$25,
+      [$$asideMenuWidthCode]: '400px',
```

Comment:
some temporary variables.
these will live in the grid classes, when we get around to those